File: 02.PYTHON/pg-166-169/pg_166_02.py
print("");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPY");
print("READ AN INTEGER AND DETERMINE IF IT HAS 3 DIGITS");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPY");
print("");


num = int(input("Write a number:  "));

if(num < 0):
    num *= -1;

if(num >=100 and num <= 999):
    print("The written number has 3 digits.");
else:
    print("The written number doesn't have 3 digits.");


"""
lastDigit = 0;
count = 0;

while(num > 0):
    lastDigit = num % 10;
    num //= 10;
    count += 1;

if(count == 3):
    print("The written number has", count, " digits.");
else:
    print("Three written number doesn't have three digits.")
"""

File: 01.JAVA/pg-166-169/pg_166_02.java
import java.util.Scanner;

public class pg_166_02
{
    public static void main(String[] args)
    {
        Scanner enter = new Scanner(System.in);
        int num, lastDigit;
        int count = 0;

        System.out.println("");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("READ A INTEGER AND DETERMINE IF IT HAS THREE DIGITS");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("");

        System.out.print("Write an integer: ");
        num = enter.nextInt();

        if(num < 0)
        {
            num *= -1;
        }

        if(num >= 100 && num <= 999)
        {
            System.out.println("The written number has 3 digits.");
        }
        else
            {
                System.out.println("The written number doesn't have 3 digits.");
            }
/*
        while(num > 0)
        {
            lastDigit = num % 10;
            num /= 10;
            count +=1;
        }

        if(count == 3)
            {
                System.out.println("The written number has " + count + " digits.");
            }
            else
                {
                    System.out.println("The written number doesn't have three digits");
                }
*/
    }
}